modules/features/widgets/src/main/kotlin/au/com/shiftyjelly/pocketcasts/widget/ui/LargePlayerQueue.kt

```diff
@@ -25,7 +25,9 @@ internal fun LargePlayerQueue(
 ) {
     val lastIndex = queue.lastIndex
     LazyColumn(
-        modifier = modifier.fillMaxWidth(),
+        modifier = modifier
+            .fillMaxWidth()
+            .height(190.dp),
```

Comment:
I considered making the height dependent on the number of elements in the list so there's no empty space at the bottom for 1 and 2 episodes. However I decided against it because then widget might seem to take too much space on the home screen while it would be eventually expanded.